phpstan-baseline.neon

```diff
@@ -1,81 +1,76 @@
 parameters:
-    ignoreErrors:
-        -
-            message: "#^Method Wallabag\\\\AnnotationBundle\\\\Controller\\\\WallabagAnnotationController\\:\\:postAnnotationAction\\(\\) should return Symfony\\\\Component\\\\HttpFoundation\\\\JsonResponse but returns Symfony\\\\Component\\\\Form\\\\FormInterface\\<mixed\\>\\.$#"
-            count: 1
-            path: src/Wallabag/AnnotationBundle/Controller/WallabagAnnotationController.php
+	ignoreErrors:
```

Comment:
tabs are the recommended indendation for neon files? If so, you should update `.editorconfig` file to add a custom rule to ensure tab are used instead of spaces.

Reply:
I don't know, I always use the `--generate-baseline` option of `phpstan analyze` to update the baseline. never edited manually, but good idea to update the `.editorconfig`